### ISSUE TITLE
[BUG] guid already exist / Synology

### ISSUE THREAD

**Issue by N1ck0-1as** [mandarons/icloud-docker#223]
**Describe the bug**
on Synology, the « users » group is 100. The container doesn’t want to pass this GUID in environment variable because it is already taken. 

With other groups like Administrators, it works but files are not available for a user in the users group.

**Comment by mandarons** [comment]
Yeah, same problem is seen on unraid as well. Looking into revamping PUID/GUID implementation.

Thanks for filling this one.

**Comment by mandarons** [comment]
Please upgrade to version 1.20.0. It should fix this problem.

**Comment by mandarons** [comment]
Stale
